src/components/tables/DataPlanes/DataPlaneDialog.tsx

```diff
@@ -0,0 +1,217 @@
+                                })}
+                                value={dataPlaneDetails.dataPlaneName.provider}
+                                showCopyButton={false}
+                            />
+                        ) : null}
+
+                        <DataPlaneDialogField
+                            label={intl.formatMessage({ id: 'data.name' })}
+                            value={
+                                dataPlaneDetails.dataPlaneName
+                                    ? formatDataPlaneName(
+                                          dataPlaneDetails.dataPlaneName
+                                      )
+                                    : '-'
+                            }
+                            showCopyButton={false}
+                        />
+
+                        <DataPlaneDialogField
+                            label={intl.formatMessage({
+                                id: 'data.internalId',
+                            })}
+                            value={dataPlane.data_plane_name}
+                        />
+
+                        {dataPlane.aws_iam_user_arn ? (
+                            <DataPlaneDialogField
+                                label={intl.formatMessage({
+                                    id: 'data.awsIamUserArn',
+                                })}
+                                value={dataPlane.aws_iam_user_arn}
+                            />
+                        ) : null}
+
+                        {dataPlane.gcp_service_account_email ? (
+                            <DataPlaneDialogField
+                                label={intl.formatMessage({
+                                    id: 'data.gcpServiceAccount',
+                                })}
+                                value={dataPlane.gcp_service_account_email}
+                            />
+                        ) : null}
+
+                        {dataPlane.data_plane_fqdn ? (
+                            <DataPlaneDialogField
+                                label={intl.formatMessage({
+                                    id: 'data.idProvider',
+                                })}
+                                value={`${OPENID_HOST}/${dataPlane.data_plane_fqdn}`}
+                            />
+                        ) : null}
+
+                        <DataPlaneDialogField
+                            label={intl.formatMessage({
+                                id: 'data.ipv4',
+                            })}
+                            value={ipv4}
+                        />
+
+                        <DataPlaneDialogField
+                            label={intl.formatMessage({
+                                id: 'data.ipv6',
+                            })}
+                            value={ipv6}
+                        />
+                    </Stack>
+                </DialogContent>
+            ) : null}
```

Comment:
Even though this _shouldn't_ happen we should still have some better handling so the user does not just open a blank dialog. 

I would show some basic `AlertBox` with a basic error.